tests/clients/test_experiment.py

```diff
@@ -503,6 +504,30 @@ def test_experiment_create_run(mocker):
     )
 
 
+def test_experiment_create_run_experiment_not_active_conflict(mocker):
+    message = "experiment not active"
+    error_code = "experiment_not_active"
+    response_mock = mocker.Mock()
+    response_mock.json.return_value = {"experimentId": "test-id"}
```

Comment:
It seems to me we don't need to mock the json return value, because we're relying on the base client to parse the JSON into the `Conflict` object.

Reply:
Sorry, I just noticed we're extracting the experiment ID... Ignore me.